onnx/onnx-ml.proto

```diff
@@ -433,8 +497,22 @@ message ModelProto {
   // One FunctionProto can reference other FunctionProto in the model, however, recursive reference
   // is not allowed.
   repeated FunctionProto functions = 25;
+
+  // Describes different target configurations for a multi-device use case.
+  // A model can describe multiple multi-device configurations for execution.
+  repeated DeviceConfigurationProto configuration = 26;
```

Comment:
Should this information belong to a graph? How are values referred to when there are subgraphs?

Reply:
Sorry I was mistaken